giskard_vision/landmark_detection/tests/base.py

```diff
@@ -169,6 +169,20 @@ def definition(prediction_result: PredictionResult, marks: np.ndarray, **kwargs)
         """
         ...
 
+    @staticmethod
+    @abstractmethod
```

Comment:
I don't think this should be an `abstractmethod`, as in it should be a hard requirement, let's make it return `None` by default.